apps/fluent-tester/src/FluentTester/TestComponents/Expander/ExpanderTest.tsx

```diff
@@ -0,0 +1,124 @@
+import * as React from 'react';
+import { Expander } from '@fluentui-react-native/experimental-expander';
+import { Text } from '@fluentui/react-native';
+import { Stack } from '@fluentui-react-native/stack';
+import { stackStyle, commonTestStyles as commonStyles } from '../Common/styles';
+import { Test, TestSection, PlatformStatus } from '../Test';
+import { EXPANDER_TESTPAGE } from './consts';
+import { View, Switch } from 'react-native';
+
+const expanderTest: React.FunctionComponent = () => {
+  /** This test page has not yet been tested and does not currently build because
+   * the react-native-test-app does not yet support WinUI 2.6
+   */
```

Comment:
I would leave a link to the RNTA issue that you opened here https://github.com/microsoft/react-native-test-app/issues/444

Thanks for adding this!